tests/functional/monitoring/prometheus/alerts/test_alert_mds_xattr_latency.py

```
@@ -0,0 +1,625 @@
+
+    while time.time() - start_time < timeout:
+        try:
+            active_mds = cluster.get_active_mds_info()
+            standby_mds = cluster.get_mds_standby_replay_info()
+
+            if active_mds and standby_mds:
+                log.info(
+                    f"MDS daemons are active: active={active_mds['mds_daemon']}, standby={standby_mds['mds_daemon']}"
+                )
+                break
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)